runtime/src/execute_cost_table.rs

```diff
@@ -89,8 +151,21 @@ impl ExecuteCostTable {
             self.prune_to(&((current_size as f64 * PRUNE_RATIO) as usize));
```

Comment:
seems like we can do this w/o floats too

Reply:
yep, did it in a separate PR #24026